### PR TITLE
Adds traits for a character's preference towards pineapples.

### PR DESCRIPTION
**Pull request by ShizCalev**

### CODE REVIEW

File: code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -12,6 +12,40 @@
...
+	species.liked_food |= PINEAPPLE
+
+/datum/trait/pineapple_liker/remove()
+	var/mob/living/carbon/human/H = trait_holder
+	var/datum/species/species = H.dna.species
+	species.liked_food &= ~PINEAPPLE
+
+/datum/trait/pineapple_hater
+	name = "Ananas Aversion"
+	desc = "You find yourself greatly detesting fruits of the ananas genius. Serious, how the hell can anyone say these things are good? And what kind of madman would even dare putting it on a pizza!?"
```
Comment (pigeonsk) [inline code/datums/traits/neutral.dm]: genus unless genius was intentional

**Comment by ShadowDeath6** [comment]
isn't this what deviant taste is for? I don't think it's a good idea to add a snowflake flag just for pineapples when the "taste trait" already exists

**Comment by PKPenguin321** [comment]
Pineapple is good though it just sucks on pizza

**Comment by AverageJoe82** [comment]
best trait available, now these pineapple fanatics can no longer force me to like their shitty pizza

**Comment by XDTM** [comment]
Can you take both traits at once?

**Comment by Tlaltecuhtli** [comment]
can you make pineapples a server config?

**Comment by Xhuis** [comment]
This is the pinnacle of traits

**Comment by Qustinnus** [comment]
sorry you used the wrong bitflag I already made a toxic bitflag specifically for this edge case.

**Review by Cyberboss — APPROVED** [review]
Every day we stray further from god's light

**Comment by Cyberboss** [comment]
@ShadowDeath6 Deviant tastes just reverses liked and disliked foods

**Comment by ShadowDeath6** [comment]
wouldn't it be better to have deviant tastes influence pinapples rather than have an entire flag called "pineapple" just for this one trait? it seems extremely snowflake and clunky

**Comment by Xhuis** [comment]
An entire flag for pineapple is something we have needed in our codebase since its inception

**Pull request merged**